README.md

```diff
@@ -155,7 +155,14 @@ python3 convert-pth-to-ggml.py models/7B/ 1
 
 When running the larger models, make sure you have enough disk space to store all the intermediate files.
 
-TODO: add model disk/mem requirements
+### Requirements
```

Comment:
```suggestion
### Disk Requirements
```

Reply:
Memory requirements are the same as disk requirements.

Reply:
```suggestion
### Memory/Disk Requirements

Since the models are entirely loaded into memory at present, you’ll need enough disk space to save them and enough RAM to load them.
```

This will change with mmap but I think it could use some clarification (and I was expecting the requirements section to list e.g. Python versions / other dependencies).

Reply:
Updated in 5d83a294d1a977b6f0d09b692fabc92bf165cad3

Reply:
Memory should be separate, like an offset. During compute, there is also the _memory_ (memory_k and memory_v), which depends on the context/npredict size (as well on type if #154 gets merged).

Reply:
We will update the readme once #154 or similar are merged.